raster/identify-raster-cell/README.metadata.json

```
@@ -0,0 +1,29 @@
+{
+    "category": "Raster",
+    "description": "\nGet the cell value of a local raster at the clicked location and display the result in a callout.",
```

I don't think we usually show the new paragraph notation here?
```suggestion
    "description": "Get the cell value of a local raster at the clicked location and display the result in a callout.",
```